packages/hash/frontend/src/pages/_document.page.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import createEmotionServer from "@emotion/server/create-instance";
+import withTwindDocument from "@twind/next/document";
+import Document, { DocumentContext } from "next/document";
+
+import twindConfig from "../../twind.config";
+import { createEmotionCache } from "../theme";
+
+class MyDocument extends Document {
+  static async getInitialProps(context: DocumentContext) {
+    const initialProps = await Document.getInitialProps(context);
+    return { ...initialProps };
+  }
```

Comment:
I'm not sure why we have this and the MUI example doesn't. If you can't think of a reason either, I think we should remove it.
```suggestion
```

Reply:
It's the default code in `_document.page.tsx` file. I will remove it since we override the getInitialProps.

Reply:
Removed in 7569409